src/editor/extensions/ext-connector/ext-connector.js

```diff
@@ -393,7 +393,7 @@ export default {
 
           const parents = svgCanvas.getParents(mouseTarget.parentNode);
 
-          if ($.inArray(svgcontent, parents) !== -1) {
+          if (parents.indexOf(svgcontent) !== -1) {
```

Comment:
Does indexOf works with DOM elements?

Reply:
indexOf change to includes. it's working